### PR TITLE
Use constants for default user and policies

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
Nice Thx :)

File: daybed/acl.py [hunk 1]
```diff
@@ -10,6 +10,19 @@
 from daybed import logger
 
 
+USER_EVERYONE = 'system.Everyone'
+USER_AUTHENTICATED = 'system.Authenticated'
```
Comment (almet) [inline daybed/acl.py:14]: These `system.*` are already provided by pyramid, why not directly use them? (they are the same as defining constants)

**Comment by almet** [comment]
I don't think using a constant is useful here. That's a concept coming from pyramid and they in some sense define the usage of this term.

If we want we can use constants rather than strings, see http://docs.pylonsproject.org/projects/pyramid/en/latest/api/security.html?highlight=everyone#constants

**Comment by almet** [comment]
I wouId suggest we backoff this commit and do it again :)

**Pull request merged**